hedyorg/hedy
# [FIX] Add warning back in

Fixes #4869 

As far as I can see, @ToniSkulj removed this warning in #4320, I suspect by accident, but if not, let me know :) 

See:
<img width="1116" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/4c8d2319-6a9f-4772-8f3b-2541417bcff3">

**How to test**
Observe that code with a space at the start runs but also throws an error:

<img width="701" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/29e2191f-dbc8-4063-91ef-ebf24a1a896e">
